client/js/components/procedure/SegmentsList/FilterFlyoutCheckbox.vue

```diff
@@ -10,24 +10,35 @@
 <template>
   <li>
     <input
-      type="checkbox"
-      :checked="checked"
-      @change="toggle"
       :id="`${instance}_id_${option.id}`"
-      :name="`${instance}_name_${option.id}`">
+      :checked="checked"
+      :name="`${instance}_name_${option.id}`"
+      type="checkbox"
+      @change="toggle">
     <label
       :class="{'weight--normal': highlight === false }"
       class="inline-block u-m-0"
       :for="`${instance}_id_${option.id}`">
       {{ option.attributes.label }} <template v-if="showCount">({{ option.attributes.count }})</template>
     </label>
+    <dp-contextual-help
+      v-if="option.attributes.description && instance !=='itemsSelected'"
+      class="float-right mt-8"
+      icon="question"
```

Comment:
The `icon` prop of DpContextualHelp defaults to `question`, so this can be removed here.

Reply:
Done here [be33bc2](https://github.com/demos-europe/demosplan-core/pull/2376/commits/be33bc2d4b3ad5fce405dfbb868049c07e0673ee)